metricbeat/module/nats/varz/_meta/data.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"nats",
+        "name":"varz",
+        "rtt":44269
+    },
+    "nats":{
+        "varz":{
+            "metrics": {
+                "uptime": "2h41m42s",
+                "now": "2018-12-28T12:33:53.026864929Z",
+                "msgs.out": 0,
+                "bytes.out": 0,
+                "total_connections": 35,
+                "server_id": "bUAdpRFtMWddIBWw80Yd9D",
+                "bytes.in": 0,
+                "remotes": 0,
+                "slow_consumers": 0,
+                "msgs.in": 0,
+                "http_req_stats.root_uri": 10,
+                "http_req_stats.connz_uri": 10,
+                "http_req_stats.routez_uri": 10,
+                "http_req_stats.subsz_uri": 10,
+                "http_req_stats.varz_uri": 10,
+                "cpu": 0,
+                "mem": 4.292608e+06,
```

Comment:
Kind of surprised it's reported like this. I assume ES does the conversion automatically?